riocli/device/model.py

```diff
@@ -79,6 +80,9 @@ def apply(self, *args, **kwargs) -> ApplyResult:
         onboard_command = onboard_script.full_command()
         execute_onboard_command(hwil_response.id, onboard_command)
 
+        if self.spec.get('virtual', {}).get('wait', False):
```

Comment:
suggestion(non-blocking): Maybe we can extract the `virtual` object before line 40.

Reply:
Done.